src/transaction/log_checkpoint_info.cpp

```diff
@@ -16,6 +16,14 @@
  *
  */
 
+#include "log_impl.h"
+#include "client_credentials.hpp"
+#include "critical_section.h"
+#include "memory_alloc.h"
+#include "transaction_global.hpp"
+#include "thread_entry.hpp"
+#include "scope_exit.hpp"
+
 #include "log_checkpoint_info.hpp"
```

Comment:
Includes format:

Include own header first.

Alphabetical order of project includes.

Alphabetical order of system includes.